File: week 3 - raindrops/raindrops.r
raindrops <- function(number) {
  drops <- ""
  
  if (number %% 3 == 0) {
    drops <- paste0(drops, "Pling")
  }
  
  if (number %% 5 == 0) {
    drops <- paste0(drops, "Plang")
  }
  
  if (number %% 7 == 0) {
    drops <- paste0(drops, "Plong")
  }
  
  if (drops == "") {
    drops <- as.character(number)
  }
  
  return(drops)
}

File: week 3 - raindrops/raindrops.rb
class Raindrops
  def self.convert(number)
    drops = ''
    
    if number % 3 == 0
      drops = drops + "Pling"
    end
    
    if number % 5 == 0
      drops = drops + "Plang"
    end
    
    if number % 7 == 0
      drops = drops + "Plong"
    end
    
    if drops == ''
      drops = number.to_s
    end
    
    drops
  end
end
